operate/Makefile

```diff
@@ -96,4 +100,6 @@ start-e2e:
 	CAMUNDA_OPERATE_CSRF_PREVENTION_ENABLED=false \
 	SERVER_PORT=8081 \
 	MANAGEMENT_SERVER_PORT=9601 \
-	mvn clean install -DskipTests -DskipChecks -f ../dist/pom.xml exec:java -Dexec.mainClass="io.camunda.application.StandaloneCamunda" -Dspring.profiles.active=operate,broker,auth
+	mvn clean install -DskipTests -DskipChecks -f ../dist/pom.xml exec:java \
```

Comment:
❓ should we align the env vars used here with `env-up:`? especially either `CAMUNDA_OPERATE_ELASTICSEARCH_CREATESCHEMA` or  `ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_ARGS_INDEX_SHOULDWAITFORIMPORTERS` as this should be relevant here too?